### ISSUE TITLE
HistoricalFoo model is defined in wrong module path

### ISSUE THREAD

**Issue by hongquan** [treyhunner/django-simple-history#343]
Given I have a project layout like this.

```
+- hatthoc/
|     +- farm/
|          +- apps.py
|          +- models.py
+- manage.py
```
Content of _hatthoc/farm/apps.py_:
```python
from django.apps import AppConfig

class FarmConfig(AppConfig):
    name = 'hatthoc.farm'
```

And in _models.py_, I have:

```python
class ControlLevel(models.Model):
    room = OneToOneField(Room, CASCADE)
    cond_type = CharField(max_length=3, choices=ConditionType.choices)
    level = SmallIntegerField(choices=ControlLevelType.choices, default=ControlLevelType.actuator)
    history = HistoricalRecords()
```

The problem is that, _django-simple-history_ creates a model `HistoricalControlLevel` and declares it to be located at `hatthoc.farm`, instead of `hatthoc.farm.models`.

This causes error when some tools like [django-extensions](https://github.com/django-extensions/django-extensions) do automatic import, like:

```sh
./manage.py shell_plus
Failed to import 'HistoricalActuatorSchedule' from 'hatthoc.farm' reason: Module "hatthoc.farm" does not define a "HistoricalActuatorSchedule" attribute/class
Failed to import 'HistoricalControlLevel' from 'hatthoc.farm' reason: Module "hatthoc.farm" does not define a "HistoricalControlLevel" attribute/class
from hatthoc.farm import
```

Note: Manual import still works

```python
from hatthoc.farm.models import HistoricalControlLevel
```

And I found out the culprit is this line https://github.com/treyhunner/django-simple-history/blob/master/simple_history/models.py#L126

```python
            else:
                models_module = app.name
            attrs['__module__'] = models_module
```
where `app.name` is `hatthoc.farm` as shown by the _apps.py_ file.

**Comment by rossmechanic** [comment]
#324 should probably fix this issue.

**Comment by rossmechanic** [comment]
This should be fixed as of version 1.9.1

**Comment by lsolberg** [comment]
I have the excact same issue on `django-simple-history==2.0`. Doesnt look like #324 made it in `2.0`..
